pkg/env/roxctl.go

```diff
@@ -1,12 +1,36 @@
 package env
 
 var (
+	// CACertFileEnv allows to pass a custom CA certificate file (path to a certificate file in PEN format).
+	CACertFileEnv = RegisterSetting("ROX_CA_CERT_FILE")
+
+	// ClientForceHTTP1Env ...
+	ClientForceHTTP1Env = RegisterBooleanSetting("ROX_CLIENT_FORCE_HTTP1", false)
+
+	// DirectGRPCEnv ...
+	DirectGRPCEnv = RegisterBooleanSetting("ROX_DIRECT_GRPC_CLIENT", false)
+
 	// EndpointEnv specifies the central endpoint to use for commandline operations.
 	EndpointEnv = RegisterSetting("ROX_ENDPOINT")
 
+	// InsecureClientEnv enables insecure client connection options (DANGEROUS, USE WITH CAUTION).
+	InsecureClientEnv = RegisterBooleanSetting("ROX_INSECURE_CLIENT", false)
+
+	// InsecureClientSkipTLSVerifyEnv allows commandline clients to skip the TLS certificate validation.
+	InsecureClientSkipTLSVerifyEnv = RegisterBooleanSetting("ROX_INSECURE_CLIENT_SKIP_TLS_VERIFY", false)
+
+	// NoColorPrinterEnv disables commandline color output.
+	NoColorPrinterEnv = RegisterBooleanSetting("ROX_NO_COLOR_PRINTER", false)
```

Comment:
Maybe name this `ROX_NO_COLOR`, since the flag is also named as such? Could be confusing for some that this suddenly is `_PRINTER`?